python/lsst/pipe/base/cli/cmd/commands.py

```diff
@@ -62,10 +63,31 @@ def register_instrument(*args: Any, **kwargs: Any) -> None:
 def transfer_from_graph(**kwargs: Any) -> None:
     """Transfer datasets from a quantum graph to a destination butler.
 
-    SOURCE is a URI to the Butler repository containing the RUN dataset.
+    SOURCE is a URI to the source quantum graph file.
 
     DEST is a URI to the Butler repository that will receive copies of the
     datasets.
     """
     number = script.transfer_from_graph(**kwargs)
     print(f"Number of datasets transferred: {number}")
+
+
+@click.command(short_help="Make Zip archive from output files using graph.", cls=ButlerCommand)
+@click.argument("graph", required=True)
+@repo_argument(required=True)
+@click.argument("dest", required=True)
+@dataset_type_option(help="Dataset types to include in Zip archive.")
+@options_file_option()
+def zip_from_graph(**kwargs: Any) -> None:
+    """Transfer datasets from a quantum graph to a destination butler.
+
+    SOURCE is a URI to the source quantum graph file to use when building the
+    Zip archive.
+
+    REPO is a URI to a butler configuration that is used to configure the
+    datastore of the quantum-backed butler.
+
+    DEST is a directory to write the Zip archive.
```

Comment:
There is something very odd about help output, I suspect there is a big mess somewhere in MWArgumentDecorator. When I print usage of this command I get this:
```
$ butler zip-from-graph -h
Usage: butler zip-from-graph [OPTIONS] GRAPH REPO DEST

  Transfer datasets from a quantum graph to a destination butler.

  REPO is the URI or path to an existing data repository root or configuration
  file.

  SOURCE is a URI to the source quantum graph file to use when building the
  Zip archive.

  REPO is a URI to a butler configuration that is used to configure the
  datastore of the quantum-backed butler.

  DEST is a directory to write the Zip archive.

Options:
...
```
Note there are two mentions of REPO with different descriptions, and docstring only has one REPO string.

Reply:
Thanks. Looks like `repo_argument` brings its own help.